### ISSUE TITLE
Visualize Recency in Audit Dashboard

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#83]
I really like the audit dashboard. I think it has lots of potential. However, it would be great to have some visual indication of recency of labels--perhaps the user could select a mode and you would switch the coloring from label type to coloring by label age or perhaps you could make it so that both time of label and label type are visualized together. 

In addition, if you are currently in an audit, would be nice to see your current location on the map or, at least, the current audit neighborhood.

![image](https://cloud.githubusercontent.com/assets/1621749/14660919/911db8ce-0677-11e6-924d-5aaa96bb6fdb.png)

**Comment by kotarohara** [comment]
Yes, I agree it can be improved. Providing your current neighborhood and recent audit data sounds like a good idea.
